README.rst

```diff
@@ -189,7 +189,8 @@ Answer the prompts with your own desired options_. For example::
     Select cloud_provider:
     1 - AWS
     2 - GCP
-    3 - None
+    3 - nginx
+    4 - None
     Choose from 1, 2, 3 [1]: 1
```

Comment:
Missing `use_nginx_for_serve_media_files`. Plus cloud provider could also be serving static file content where nginx is not doing this.

Reply:
It will be after select cloud provider.